private_gpt/settings/settings.py

```diff
@@ -217,6 +217,31 @@ class QdrantSettings(BaseModel):
     )
 
 
+class SQLDatabaseSettings(BaseModel):
+    enabled: bool = Field(
+        False,
+        description="Flag to enable SQL Query mode. Disabled by default",
+    )
+    dialect: Literal["mysql"]
+    driver: Literal["pymysql"]
+    db_host: str | None = Field(
+        "localhost",
+        description="Host name of Database server. If host is None, set to 'localhost'.",
+    )
+    db_user: str | None = Field(
+        "root",
+        description="Username to be used for accessing the SQL Database Server. If user is None, set to 'root'.",
+    )
```

Comment:
The default should be left to `None`, as `root` is not a standard default user in SQL norm (actually, SQL does not tell anything on a default user, but implementations are usually defining some, e.g. `postgres` for postgreSQL).

Reply:
MySQL default admin user is called `root`.

Regardless, wrt supporting other SQL databases, I'm making username and password both default to `None`